Flutter/pdf-viewer/right-to-left.md

```diff
@@ -120,4 +120,6 @@ Then, import the `flutter_localizations` library, specify [localizationsDelegate
   }
 
 {% endhighlight %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+N> RTL scrolling is not supported in Single Page Layout mode when using horizontal scroll direction.
```

Comment:
Can this be changed to?

RTL scrolling is not supported in single page layout mode when the scroll direction is set to horizontal.